### DISCUSSION TITLE
more flexible white/black lists

### DISCUSSION THREAD

**Discussion by chupacabra71** [Luligu/matterbridge-hass#15] [Q&A]
Currently we have to add individual devices to the white or blacklist.  it would be nice to be able to include/exclude by domain, tag, wildcard, etc.

**Comment by wolfwander** [comment]
Or label. It could be great just marking a label to an entity to expose it to Matterbridge

**Reply by robvanoostenrijk** [reply]
I have a PR in the works for this. But in the current dev branch device and entity white/blacklist are marked as being removed in Matterbridge 1.6.6.

So I'm holding off until I can see what the 1.6.6 logic devised by @Luligu will be and then look at label filtering.

**Comment by Luligu** [comment]
Published under tag dev. In the frontend install matterbridge-hass@dev

## [0.0.4-dev.2] - 2024-12-12

### Added

- [homeassistant]: Add the possibility to white and black list a device with its name or its device id.
- [homeassistant]: Add the possibility to black list one or more device entities with their entity id globally or on a device base.
- [homeassistant]: Add sensor domain with temperature, humidity, pressure and illuminance.

### Changed

- [package]: Requires Matterbridege 1.6.6.
- [package]: Update dependencies.

<a href="https://www.buymeacoffee.com/luligugithub">
  <img src="./yellow-button.png" alt="Buy me a coffee" width="120">
</a>

**Comment by Luligu** [comment]
> Currently we have to add individual devices to the white or blacklist. it would be nice to be able to include/exclude by domain, tag, wildcard, etc.

Excluding a device by domain can be a really interesting idea. 
The new method allows it passing also the device domain.
